bin/MadGraph5_aMCatNLO/cards/production/13TeV/exo_diboson/Spin-0/VBFToRadionToHH_htatahbb/Template/VBFToRadionToHH_htatahbb_MMASS_proc_card.dat

```diff
@@ -0,0 +1,11 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+import model heft_radion
+define q = u c d s b u~ c~ d~ s~ b~
+define p = u c d u~ c~ d~
+generate p p > h2 q q  / g h , ( h2 > H H, H > ta+ ta-, H > b b~)
```

Comment:
The process with "/ g h" is unsafe in terms of gauge invariance and might lead to unphysical results.

<img width="736" alt="image" src="https://user-images.githubusercontent.com/17442656/129150802-e166aa2e-88f6-421c-9e7e-e677d5449220.png">
If this was used to only take VBF diagrams (bottom) you also need to realize that you still get non-VBF diagrams (above).
So I think it would be much more safe/correct thing to do if you give WBF or mmjj cuts instead of using "/ g h". And this is more efficient as in your analysis level you might require deta(jj) or m(jj)

Reply:
This was taken from an example gridpack that was intended as an example for the lower mass analysis, but they upon review they warned us that some diagrams were incorrect. The `/ g h` has been removed in the latest commits